### ISSUE TITLE
Emojis used as classnames in scss won't compile. 

### ISSUE THREAD

**Issue by ryanlabar** [Shopify/slate#241]
### Problem

I tried using an emoji as a classname and the scss wouldn't compile.

### Replication steps

Use an emoji in an scss file. 

`.😃{background:chartreuse; color:yellow;}`

### More Information

😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡🍕

**Comment by DanArpoika** [comment]
I also would like to see a fix.

**Comment by jonathanmoore** [comment]
**See: https://github.com/Shopify/slate/issues/204**

This doesn't have anything to do with CSS comment blocks, Slate or even Shopify's SCSS compiler. The solution would be to add @charset "UTF-8"; to the very top of your file to tell the SCSS compiler that you are intentionally using UTF-8 characters. Once you do that, you can go wild with UTF-8.

```css
@charset "UTF-8";
.🤑:after {
  display: block;
  content: '🎉';
}
```

**Comment by ryanlabar** [comment]
Brilliant!

**Comment by ryanlabar** [comment]
Even after adding `@charset "UTF-8";` the compiler fails as soon as I add an Emoji. I added that to the theme.scss file. Any Suggestions?

**Comment by t-kelly** [comment]
@ryanlabar could you post the code snippet you are describing?

**Comment by t-kelly** [comment]
@ryanlabar I just tested this fix in #242. It seems to have worked for me. Are you sure there is not another formatting issue in your stylesheet?

![image](https://user-images.githubusercontent.com/4837696/29569405-69359026-8721-11e7-8553-6b1ce09f035e.png)

**Comment by t-kelly** [comment]
PR has been merged. Will ship in the 0.12.0 release of Slate.
